src/ui/public/vislib/visualizations/_map.js

```diff
@@ -259,16 +259,23 @@ export default function MapFactory(Private, tilemap, $sanitize) {
         if (NWlng < -180) {
           NWlng += 360;
         }
+
+        const SELat = bounds.getSouthEast().lat;
+        const NWLat = bounds.getNorthWest().lat;
+        if (SELat === NWLat || SElng === NWlng) {
+          return;
```

Comment:
Suggestion 1: Can we add a comment to explain this? E.g. `Bounds can't be created unless they form a box.`

Suggestion 2L I know this expands the scope of this PR _very slightly_ so it's up to you if you want to go for it. But I think it will be easier to follow this code if we used the vars `northWestLat`, `northWestLng`, `southEastLat`, and `southEastLng`. I'm sure once you stare at this long enough the current vars become transparent but as I was reviewing I found myself having to mentally convert SE to south-east and NW to north-west to reason through the code.